galley/pkg/server/server.go

```diff
@@ -229,6 +238,9 @@ func (s *Server) Run() {
 			scope.Fatalf("Galley Server unexpectedly terminated: %v", err)
 		}
 	}()
+	if s.callOut != nil {
+		go s.callOut.Run()
```

Comment:
Can you reconsider leaking goroutines? Blocking functions are easier to understand and test, eg. look at the trouble it brought in https://github.com/istio/istio/pull/11335/files